src/modules/extendedMap/settings.ts

```diff
@@ -90,7 +90,7 @@ export default <ModuleSettingFunction>(async (MODULE_ID, LSSM, $m) => {
     userBuildings
         .map(({ id, caption, building_type }) => [
             id.toString(),
-            `[${buildingTypes[building_type].caption}] ${caption}`,
+            `[${buildingTypes[building_type]?.caption ?? '🌈'}] ${caption}`,
```

Comment:
I really appreciate the emojis, but we should use a more descriptive emoji to tell the user that something went wrong like `⚠️` or `‼️`